server/src/test/java/org/apache/druid/server/coordinator/duty/KillUnusedSegmentsTest.java

```diff
@@ -644,6 +648,26 @@ public void testKillSecondHalfEternitySegment()
     validateLastKillStateAndReset(DS1, secondHalfEternity);
   }
 
+  @Test
+  public void testKillLargeIntervalSegments()
+  {
+    final Interval largeTimeRange1 = Intervals.of("1990-01-01T00Z/19940-01-01T00Z");
```

Comment:
Are these intervals just meant to be large or half-eternity intervals?

Reply:
This is meant to be large. The naming convention follows `IndexerSQLMetadataStorageCoordinatorTest`. The unit test for half eternity is `testKillSecondHalfEternitySegment`.